### DISCUSSION TITLE
DataNode 老年代使用率高，是否存在影响

### DISCUSSION THREAD

**Discussion by luxin763323387** [apache/iotdb#9757] [Q&A]
<img width="823" alt="image" src="https://user-images.githubusercontent.com/38853540/236127894-f1f25a18-8357-420e-a6a1-1aa318fd6d7f.png">

单机版本部署后第10天，凌晨的时候出现过oom。后重启数据库，进行对jvm内存进行监控
dataNode 检测jvm 堆内存使用率经常突破80%，old老年代使用率正常吗？
iotdb版本为1.1.0，使用的是单机版本模式

**Comment by luxin763323387** [comment]
<img width="850" alt="image" src="https://user-images.githubusercontent.com/38853540/236128620-5fe94445-6888-4311-8c1f-64f2a8fcee9a.png">
使用的是Java11, CGC后，old 使用率仍然很高。这是什么原因？

**Comment by HTHou** [comment]
试试 dump 内存看下是哪些对象一直没有释放？
